kubectl-fdb/cmd/plugin_version_checker.go

```diff
@@ -0,0 +1,128 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/go-retryablehttp"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+// VersionChecker interface to help us mock test the version checker
+type VersionChecker interface {
+	getLatestPluginVersion() (string, error)
+}
+
+// RealVersionChecker will do actual call to GitHub API to get version
+type RealVersionChecker struct{}
+
+// KubectlFbReleaseURL is the public GitHub URL we read the latest version from
+const KubectlFbReleaseURL = "https://api.github.com/repos/FoundationDB/fdb-kubernetes-operator/releases/latest"
+
+// LocalTempVersionFileName is where we cache plugin version for 24 hours, so we don't call GitHub for every single command(also rate-limit issue on GitHub api calls)
+const LocalTempVersionFileName = "latest.plugin"
```

Comment:
```suggestion
const LocalTempVersionFileName = "kubectl-fdb-version.txt"
```

`latest.plugin` is a bit generic and I would prefer to have a more specific name.